google/cloud/storage/tools/make_jwt_assertion_for_test_data.py

```diff
@@ -0,0 +1,83 @@
+"""Create a JWT assertion for test purposes.
+
+Requires package `python-jose`. This script was written/tested on Python 2.7.
+
+Usage: Change any parameters in the contents dictionary below as necessary, then
+simply run this script with no arguments to dump the JWT assertion string.
+"""
+
+from jose import jwk
+from jose import jws
+
+
+# pylint: disable=line-too-long
+############################################
+# These values are copy/pasted from the values used in our tests.
```

Comment:
Maybe: "make sure the values here are identical to the values in service_account_credentials_test.cc"? Alternatively, you could change this script to create a `.inc` file that is included by the test, I think there is an example of that in `google/cloud/bigtable/tools/`.

Reply:
Opted for just leaving the comment.  I don't think we'll have to change this terribly often, but if that ends up being the case, I'll work on making this less of a copy/paste process.